### PR TITLE
Migration fix for `tooling` branch

### PR DESCRIPTION
**Pull request by adamdossa**
Fix migration code and remove bit masking.

### CODE REVIEW

File: primitives/src/cdd_id.rs [hunk 1]
```diff
@@ -34,7 +34,7 @@ impl InvestorUid {
 impl From<[u8; 32]> for InvestorUid {
     /// Ensure DID < 2^255 by masking the high bit, that facilitates its conversion to Scalar.
```
Comment (satyamakgec) [inline primitives/src/cdd_id.rs:35]: we can remove the comment as well.

**Review by Centril — APPROVED** [review]

File: primitives/src/cdd_id.rs (hunk 1 above)
Comment (satyamakgec) [reply primitives/src/cdd_id.rs:35]: We already have some code for this in develop we can use that

```
fn from(s: [u8; 32]) -> Self {
        let mut short: [u8; 16] = Default::default();
        short.copy_from_slice(&s[..16]);
        Self(short)
    }
```

**Review by satyamakgec — APPROVED** [review]

File: primitives/src/cdd_id.rs [hunk 2]
```diff
@@ -34,7 +34,7 @@ impl InvestorUid {
...
     fn from(mut s: [u8; 32]) -> Self {
-        s[31] &= 0b0111_1111;
+        //s[31] &= 0b0111_1111;
```
Comment (maxsam4) [inline primitives/src/cdd_id.rs:37]: Dead code can be removed and the comment can be updated.

File: primitives/src/identity_id.rs [hunk 3]
```diff
@@ -157,7 +157,7 @@ impl From<[u8; UUID_LEN]> for IdentityId {
 impl IdentityId {
     /// Ensure DID < 2^255 by masking the high bit, that facilitates its conversion to Scalar.
     pub const fn from_bytes(mut s: [u8; UUID_LEN]) -> Self {
-        s[UUID_LEN - 1] &= 0b0111_1111;
+        //s[UUID_LEN - 1] &= 0b0111_1111;
```
Comment (maxsam4) [inline primitives/src/identity_id.rs:160]: Dead code can be removed and the comment can be updated.

**Review by maxsam4 — APPROVED** [review]

**Pull request merged**